Main.py

```diff
@@ -693,7 +693,7 @@ def on_flash(event):
                     if self._config.adb_id:
                         # Make sure Phone model matches firmware model
                         if self._config.firmware_model != self._config.adb_model:
-                            print("ERROR: Android Device Model %s does not match Firmaware Model %s" % (self._config.adb_model, self._config.firmware_model))
+                            print("ERROR: Android device model %s does not match firmaware model %s" % (self._config.adb_model, self._config.firmware_model))
```

Comment:
I think we both missed a typo here :)
firmware, the others on this suggestion need to be in lowercase as you suggested.